text/supportlettre.scad

```diff
@@ -0,0 +1,19 @@
+// Support text
```

Comment:
What's the purpose of this test? There are already overhang and part planning tests with this sort of shape. The other tests in this category are meant for labelling things. Like, to label a build plate when sending them via wifi to a cluster of printers, or to label the layer numbers.

This test does not serve the same purpose, since the text will most likely fail to print.